spec/chartmogul/account_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe ChartMogul::Account, uses_api: true do
+  describe 'API interactions', vcr: true, record: :all do
+    let(:account) { ChartMogul::Account.retrieve }
+
+    it 'returns the name of current account', uses_api: true do
```

Comment:
Can you please record just one VCR, ie. make it use the same one?

Reply:
Done!